### ISSUE TITLE
feature request: make the news app use rows rather than columns

### ISSUE THREAD

**Issue by ckuethe** [Othernet-Project/RxOS-UI2#21]
TL;DR: shuffle the news UI to have a full width scroll item of news providers; below that, a full width scroll item of news articles; below that, the article.

The long version:  columnar layout in the news app isn't really great on mobile. On a desktop browser it looks ok:

![Screenshot_2019-04-02_12-25-48](https://user-images.githubusercontent.com/1131751/55430554-42005a80-5543-11e9-8757-6669ed6954b8.png)

But on a phone it's difficult to see the the news providers or the article titles:
![Screenshot_20190402-122608_Chrome](https://user-images.githubusercontent.com/1131751/55430603-61978300-5543-11e9-862b-08cfeb8c0eff.jpg)

It's still not great in landscape mode:
![Screenshot_20190402-122642_Chrome](https://user-images.githubusercontent.com/1131751/55430628-770cad00-5543-11e9-93e9-21573d05a32e.jpg)
